@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -217,6 +223,27 @@ export class PickHandler {
         return pickListener.results;
     }
 
+    /**
+     * Returns a ray caster using the supplied screen positions.
+     *
+     * @param x - The X position in css/client coordinates (without applied display ratio).
+     * @param y - The Y position in css/client coordinates (without applied display ratio).
+     *
+     * @alpha
```

Comment:
I think you can make it private

Reply:
I can't, TerrainElevationProvider uses that, which I forgot to change btw. Shall I mark it as internal?

Reply:
ahhh, sorry I did only a search in coresdk, my bad.

yes, please mark it as internal then

Reply:
actually the whole PickHandler is internal, so no need.